jakarta-eclipse/org.eclipse.lsp4jakarta.core/src/org/jakarta/codeAction/CodeActionHandler.java

```diff
@@ -95,10 +99,9 @@
                         || diagnostic.getCode().getLeft().equals(ServletConstants.DIAGNOSTIC_CODE_FILTER_DUPLICATE_ATTRIBUTES)) {
                         codeActions.addAll(CompleteFilterAnnotationQuickFix.getCodeActions(context, diagnostic, monitor));
                     }
-                    if(diagnostic.getCode().getLeft().equals(PersistenceConstants.DIAGNOSTIC_CODE_INVALID_ANNOTATION)) {
-                    	codeActions.addAll(DeleteConflictMapKeyQuickFix.getCodeActions(context, diagnostic, monitor));
```

Comment:
Looks like the persistence test is failing because the associated code action has been removed - add this back in